home/templates/home/tags/image.html

```diff
@@ -0,0 +1,25 @@
+{% load static wagtailcore_tags wagtailimages_tags %}
+
+{% if width == 360 %}
```

Comment:
Is there no other way to do this than to have a condition for every possible width?

Reply:
Basically, wagtail's image tag can't handle variables and it needs a constant string for filter specs, for reference see [here](https://github.com/wagtail/wagtail/issues/2090).
Wagtail has made some progress around it but is not quite there yet to handle this completely.
For now, we are doing this through if...elif...else instead of monkey patching wagtail image tag
Furthermore, instead of doing this in multiple places, we wrote an abstraction over the wagtail's image tag to hide away this ugly if...elif...else structure.
Let me know your thoughts on this.